resources/schema.json

```diff
@@ -0,0 +1,1990 @@
+          },
+          "type": "array",
+          "title": "SMS Notification",
+          "description": "SMS Notification Configuration"
+        },
+        "teams": {
+          "items": {
+            "$ref": "#/$defs/notify_teams_NotifyConfig"
+          },
+          "type": "array",
+          "title": "Teams Notification",
+          "description": "Teams Notification Configuration"
+        },
+        "shell": {
+          "items": {
+            "$ref": "#/$defs/notify_shell_NotifyConfig"
+          },
+          "type": "array",
+          "title": "Shell Notification",
+          "description": "Shell Notification Configuration"
+        }
+      },
+      "additionalProperties": false,
+      "type": "object"
+    },
+    "notify_aws_Credentials": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "title": "AWS Access Key ID",
+          "description": "AWS Access Key ID"
+        },
+        "key": {
+          "type": "string",
+          "title": "AWS Access Key Secret",
+          "description": "AWS Access Key Secret"
+        }
+      },
+      "additionalProperties": false,
+      "type": "object",
+      "required": [
+        "id",
+        "key"
+      ]
+    },
+    "notify_aws_NotifyConfig": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "title": "Notification Name",
+          "description": "The name of the notification"
+        },
+        "channels": {
+          "items": {
+            "type": "string"
+          },
+          "type": "array",
+          "title": "Notification Channels",
+          "description": "The channels of the notification"
+        },
+        "dry": {
+          "type": "boolean",
+          "title": "Dry Run",
+          "description": "If true the notification will not send the message"
+        },
+        "timeout": {
+          "type": "integer",
+          "title": "Timeout",
+          "description": "The timeout of the notification"
+        },
+        "retry": {
+          "$ref": "#/$defs/global_Retry",
+          "title": "Retry",
+          "description": "The retry of the notification"
+        },
+        "region": {
+          "type": "string",
+          "title": "AWS Region ID",
+          "description": "AWS Region ID",
+          "examples": [
+            "\"us-west-2\""
+          ]
+        },
+        "endpoint": {
+          "type": "string",
+          "title": "AWS Endpoint",
+          "description": "AWS Endpoint",
+          "examples": [
+            "\"https://sns.us-west-2.amazonaws.com\""
+          ]
+        },
+        "credential": {
+          "$ref": "#/$defs/notify_aws_Credentials",
+          "title": "AWS Credential",
+          "description": "AWS Credential"
+        },
+        "profile": {
+          "type": "string",
+          "title": "AWS Profile",
+          "description": "AWS Profile"
+        },
+        "format": {
+          "type": "integer",
+          "enum": [
```

Comment:
All values of the enum both are `0`? It looks something was wrong.

Reply:
thanks, I will fix this